common/flatpak-context-private.h

```diff
@@ -83,10 +83,24 @@ extern const char *flatpak_context_devices[];
 extern const char *flatpak_context_features[];
 extern const char *flatpak_context_shares[];
 
-gboolean       flatpak_context_parse_filesystem (const char             *filesystem_and_mode,
-                                                 char                  **filesystem_out,
-                                                 FlatpakFilesystemMode  *mode_out,
-                                                 GError                **error);
+typedef enum {
+  FLATPAK_CONTEXT_FILESYSTEM_FLAGS     = 0,
+  FLATPAK_CONTEXT_FILESYSTEM_FLAGS_ALL =  1 << 0,
```

Comment:
#4678 uses `:reset` and `_RESET`. I think that name is better than `:all`, but I'm still not particularly happy with it; for now, I'm using it as a distinctive placeholder until I can get the implementation right. It will be reasonably straightforward to replace with something better if we can think of something better.